client/src/Signup/Signup.js

```diff
@@ -92,286 +75,305 @@ export function Signup() {
-                errors={errors.phoneNumber}
-                inputId="phoneNumber"
-                onInput={event => {
-                  // TODO: refactor this into a reusable masker?
-                  // masks input to US phone number format
-                  var x = event.target.value
-                    .replace(/\D/g, '')
-                    .match(/(\d{0,3})(\d{0,3})(\d{0,4})/)
-                  event.target.value = !x[2]
-                    ? x[1]
-                    : '(' + x[1] + ') ' + x[2] + (x[3] ? '-' + x[3] : '')
                   setUser({ ...user, phoneNumber: event.target.value })
-                }}
-                placeholder="(888) 888-8888"
-                register={register({
-                  pattern: {
-                    value: /(?:\d{1}\s)?\(?(\d{3})\)?-?\s?(\d{3})-?\s?(\d{4})/,
-                    message: 'Please provide a valid phone number.'
-                  }
-                })}
-                showValidationError={false}
-                type="tel"
+                }
               />
-            </div>
-            {/*
-              places validationError on the parent component so we don't get
-              multiple error messages or misplaced messages; the phoneType box should
-              be highlighted like an error if the phoneNumber is invalid, but
-              the message only needs to be displayed once for the whole "fieldset"
-             */}
-            {errors.phoneNumber && (
-              <ValidationError errorMessage={errors.phoneNumber.message} />
-            )}
-          </div>
+            </Form.Item>
+          </Input.Group>
+        </Form.Item>
 
-          <ToggleInput
-            errors={errors.language}
-            inputId="language"
-            label="Preferred language"
-            labelClasses="mb-2"
+        <Form.Item
+          label="Preferred Language"
+          name="language"
+          valuePropName="checked"
+          // explicity styling around Ant's strong "width of radio buttons" opinion
+          className="mb-0"
+          style={{ marginBottom: '-6px' }}
+          rules={[
+            { required: true, message: 'Preferred language is required' }
+          ]}
+        >
+          <Radio.Group
+            value={user.language}
+            optionType="button"
+            buttonStyle="solid"
+            className="w-full"
             onChange={event =>
               setUser({ ...user, language: event.target.value })
             }
-            options={[
-              {
-                label: 'English',
-                value: 'en'
-              },
-              {
-                label: 'Español',
-                value: 'es'
-              }
-            ]}
-            register={register({ required: 'Language is required' })}
-            required
-            selectClasses="grid-cols-2"
-            selectedOption={user.language}
-          />
+          >
+            <Radio.Button value="en" className="w-1/2">
```

Comment:
Hey - text needs still needs to be centered (according to my last talk with Rebecca. Adding `style={{ textAlign: 'center' }}` after w-12 class name does this. Same with the Spanish button!